substrate/client/chain-spec/src/chain_spec.rs

```diff
@@ -784,9 +784,7 @@ fn json_eval_value_at_key(
 	path: &mut VecDeque<&str>,
 	fun: &dyn Fn(&json::Value) -> bool,
 ) -> bool {
-	let Some(key) = path.pop_front() else {
-		return false;
-	};
+	let Some(key) = path.pop_front() else { return false };
```

Comment:
None of our business as well. It may be wrong indeed, or it may be formatter on your side, doesn't matter, we shouldn't introduce totally unrelated changes to a PR, even that small.